gruntfile.js

```diff
@@ -461,7 +456,7 @@ module.exports = function (grunt) {
         command: [
           'docker stop portainer',
           'docker rm portainer',
-          'docker run --privileged -d -p 9000:9000 -v /tmp/portainer:/data -v /var/run/docker.sock:/var/run/docker.sock --name portainer portainer --no-analytics'
+          'docker run --privileged -d -p 9000:9000 -v $(pwd)/dist/public:/public -v /tmp/portainer:/data -v /var/run/docker.sock:/var/run/docker.sock --name portainer portainer --no-analytics'
```

Comment:
@1138-4EB suggested that we could simply move all files from `dist` folder to a subfolder rather than copying it to the root.